### PR TITLE
Fix style issues

### PR DESCRIPTION
**Pull request by nyorain**
Inspired by the many cleanup contributions in the last days i fixed some (often self-caused) code style issues.

### CODE REVIEW

File: types/wlr_keyboard.c [hunk 1]
```diff
@@ -11,7 +11,10 @@ void wlr_keyboard_init(struct wlr_keyboard *kb,
 }
 
 void wlr_keyboard_destroy(struct wlr_keyboard *kb) {
-	if (!kb) return;
+	if (!kb) {
+		return;
+	}
```
Comment (SirCmpwn) [inline types/wlr_keyboard.c]: Better to update the if statement below to

```c
if (kb && kb->impl && kb->impl->destroy)
```

imo

**Comment by martinetd** [comment]
(Heh, the failures are connexion timeout to pkgbuild.com... No idea how to get it to rerun but the site looks down for me as well atm)

That aside, do we want travis to run some checkpatch to catch new style issues?

**Comment by nyorain** [comment]
That would be nice imo. Do you know a style-checking bot for c?

**Comment by martinetd** [comment]
Not really any great fit for us - I've used checkpatch.pl (kernel's) but some rules are hard-coded for kernel style and don't fit us (e.g. they don't want optional braces like we have here, and there is no option to change that)

Some other project use astyle but I can't get it to ignore some of our quirks (like case foo: bar; break; in a single line), and it doesn't understand patch format so would need to fix everything in one go before we start using it. (it's easy to make travis complain if we have any file that changed, but not so much to tell if it was because of our patch)
I've also seen indent being used, it has a gazillion options so probably something that would match our current style.

I'm not fussy, will play with indent options for a while longer and if it's not a perfect fit I can try to hammer something into checkpatch.pl ; unless you know other alternatives?

**Comment by SirCmpwn** [comment]
I investigated automated style enforcement for Sway with clang-format and found it unsuitable for our needs.

File: types/wlr_tablet_pad.c [hunk 2]
```diff
@@ -13,7 +13,10 @@ void wlr_tablet_pad_init(struct wlr_tablet_pad *pad,
 }
 
 void wlr_tablet_pad_destroy(struct wlr_tablet_pad *pad) {
-	if (!pad) return;
+	if (!pad) {
+		return;
+	}
```
Comment (SirCmpwn) [inline types/wlr_tablet_pad.c]: Merge this with the next condition

File: types/wlr_input_device.c [hunk 3]
```diff
@@ -23,7 +23,10 @@ void wlr_input_device_init(struct wlr_input_device *dev,
 }
 
 void wlr_input_device_destroy(struct wlr_input_device *dev) {
-	if (!dev) return;
+	if (!dev) {
```
Comment (SirCmpwn) [inline types/wlr_input_device.c]: Merge with next condition

Comment (nyorain) [inline types/wlr_input_device.c]: not really possible in this case since this function does more stuff

Comment (SirCmpwn) [inline types/wlr_input_device.c]: My bad, the diff only expands up to the end of the if statement on GitHub.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**